Vates/VatesAPI/src/vtkMDHexFactory.cpp

```diff
@@ -120,8 +117,7 @@ void vtkMDHexFactory::doCreate(
       API::IMDNode *box = boxes[i];
       Mantid::signal_t signal_normalized = (box->*normFunction)();
 
-      if (std::isfinite(signal_normalized) &&
-          m_thresholdRange->inRange(signal_normalized)) {
```

Comment:
I'll time a few workspaces and see what the difference is.

Generating a vtkUnstructuredGrid from a MDEventWorkspace is fast enough that it would take a lot of boxes to make a difference. I'm more optimistic about vtkMDHistoHexFactory, especially if we can later parallelize some of those loops.